<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Librer√≠a</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Libros</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QPushButton" name="push_libro_mostrar">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_libro_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="push_libro_eliminar">
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="push_libro_nuevo">
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="edit_libro_buscar">
          <property name="placeholderText">
           <string>Buscar Libro...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableWidget" name="table_libro"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Existencia</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QLineEdit" name="edit_existencia_buscar">
          <property name="placeholderText">
           <string>Buscar Libro...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="push_existencia_mostrar">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="push_existencia_nuevo">
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableWidget" name="table_existencia"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_existencia_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Autores</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="2" colspan="2">
         <widget class="QPushButton" name="push_autor_eliminar">
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="edit_autor_buscar">
          <property name="placeholderText">
           <string>Buscar autor...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="edit_autor_guardar">
          <property name="placeholderText">
           <string>Nuevo autor...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="push_autor_mostrar">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QPushButton" name="push_autor_guardar">
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QPushButton" name="push_autor_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QTableWidget" name="table_autor"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Editoriales</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="push_editorial_eliminar">
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="push_editorial_nuevo">
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="edit_editorial_buscar">
          <property name="placeholderText">
           <string>Buscar...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="push_editorial_mostrar">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_editorial_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableWidget" name="table_editorial"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Genero</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLineEdit" name="edit_genero_buscar">
          <property name="placeholderText">
           <string>Buscar Genero..</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_genero_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QPushButton" name="push_genero_eliminar">
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="push_genero_mostrar">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="edit_genero_nuevo">
          <property name="placeholderText">
           <string>Nuevo Genero..</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QPushButton" name="push_genero_guardar">
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QTableWidget" name="table_genero"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Ventas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QDateEdit" name="date_venta_buscar">
          <property name="displayFormat">
           <string>d/M/yyyy</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="push_venta_mostrar">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="push_venta_nuevo">
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QPushButton" name="push_venta_eliminar">
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_venta_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTableWidget" name="table_venta"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Empleados</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="2">
         <widget class="QPushButton" name="push_empleado_mostrar">
          <property name="text">
           <string>Empleados</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_empleado_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QTableWidget" name="table_empleado"/>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="push_supervisor_mostrar">
          <property name="text">
           <string>Supervisores</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="edit_empleado_buscar">
          <property name="placeholderText">
           <string>Buscar empleado...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="push_empleado_nuevo">
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="push_empleado_inactivos">
          <property name="text">
           <string>Inactivos</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="2">
         <widget class="QPushButton" name="push_empleado_eliminar">
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Sucursales</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="3">
         <widget class="QPushButton" name="push_sucursal_mostrar">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3" colspan="2">
         <widget class="QPushButton" name="push_sucursal_eliminar">
          <property name="text">
           <string>Eliminar seleccionado</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QTableWidget" name="table_sucursal"/>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="edit_sucursal_buscar">
          <property name="placeholderText">
           <string>Buscar sucursal..</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="push_sucursal_buscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="push_sucursal_nuevo">
          <property name="text">
           <string>Nuevo</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="push_sucursal_inactiva">
          <property name="text">
           <string>Inactivas</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
